app/views/layout.erb

```diff
@@ -24,6 +24,15 @@
 
         </div>
     </header>
+    <% if flash[:booked] && !flash[:booked].empty? %>
```

Comment:
Is a distinct `<ul id="booked">` necessary? I guess because we would not want to style the message the same as, say, invalid form data alerts (in `<ul id="errors">`)?